### PR TITLE
#5 #99 make node_modules independent

### LINKED ISSUE #99: Disable folder sync for vendor & node_modules between host<->docker
remove the synchronisation between the host and the docker environment for the following folders:
- [x] `node_modules`
- [x] `vendor`

this way it should solve a problem when using `npm run watch` command

PR #100

### PR DESCRIPTION
**Pull request by Serphentas**

### CODE REVIEW

**Review by fabio10000 — CHANGES_REQUESTED** [review]
Ok for node_modules, but all points from de issue #99 are not covered, please refer to the issue to complete all points before the PR

**Comment by Serphentas** [comment]
See fbaf477 @fabio10000, third bullet point is not doable without an ugly script

File: infra/dev/docker-compose.yml [hunk 1]
```diff
@@ -19,6 +24,8 @@ services:
     container_name: dev_laravel
     volumes:
       - ../../src:/var/www
+      - node_modules:/var/www/node_modules/
+      - vendor:/var/www/vendor/
```
Comment (fabio10000) [inline infra/dev/docker-compose.yml:28]: This only works one way Host->Docker
But the sync from Docker->Host still working

**Review by fabio10000 — APPROVED** [review]
Not 100% working as asked on the issue but should resolve the problem

File: infra/dev/docker-compose.yml (hunk 1 above)
Comment (Serphentas) [reply infra/dev/docker-compose.yml:28]: As per [documentation](https://docs.docker.com/storage/volumes) it's not possible to have a subfolder of host be excluded from sync. Nearest option is RO from Docker, or tmpfs but changes are lost at each reboot.

Comment (Serphentas) [reply infra/dev/docker-compose.yml:28]: See [ŧhis](https://stackoverflow.com/a/55962619/2828700) also

**Pull request closed without merging**